src/resources/transactions.js

```diff
@@ -45,8 +46,9 @@ class Transactions {
         throw new SDKError(err.error_type, err.message)
       }
     }
-    if (!this.accountId) {
-      throw new SDKError(400, '[Zabo] Not connectec. See: https://zabo.com/docs#get-a-specific-transaction')
+
+    if (!this.accountId || !uuidValidate(this.accountId, 4)) {
```

Comment:
This uuid check seems to be either redundant or not catching the right error. If the error is not connected, then the presence of an accountId should always be valid because we're populating this.